draft-mouris-cfrg-mastic.md

```diff
@@ -1433,7 +1596,7 @@ described in {{Section 8 of !VDAF}}, except that the threshold represents a
 minimum weight rather than a minimum count. In addition:
 
 1. The Aggregators MUST perform the range check (i.e., verify the FLP) at the
-   first round of aggregation and remove any invalid reports before proceeding.
+first round of aggregation and remove any invalid reports before proceeding.
```

Comment:
we need the indent here to keep the enumeration

Reply:
Does it affect rendering? https://jimouris.github.io/draft-mouris-cfrg-mastic/nekt/draft-edits-2/draft-mouris-cfrg-mastic.html